docs/aerodromes/Launceston.md

```diff
@@ -47,24 +49,29 @@ Aircraft planned via **IRSOM**, **ONAGI**, **NUNPA**, **MOTRA**, **IRONS**, **MO
 
 ## Coordination
 ### Departures
-A next call is made for all aircraft when they are next to depart and will be departing within two minutes. LTA will respond by either acknowledging the callsign or offering a higher level. Any higher level issued will be passed along in the takeoff clearance. If the controller is in a high workload environment, a next call may be omitted, in favour of taxi and departure coordination.
+"Next" Coordination to LTA is required for all deps not assigned a SID.
+
+"Next" Coordination is a procedure where the **LT ADC** controller gives a heads-up to the LTA controller about an impending departure not on a SID. The LTA controller will respond by assigning a visual heading to the aircraft, for the **LT ADC** controller to pass on with their takeoff clearance.
 
 !!! example
-    **LT ADC** -> **LTA**: "Next QFA400"  
-    **LTA** -> **LT ADC**: "QFA400, FL140"  
-    **LT ADC** -> **LTA**: "FL140, QFA400"  
+    **LT ADC** -> **LT TCU**: "Next, ABC"  
+    **LT TCU** -> **LT ADC**: "ABC, Heading 150 Visual"  
+    **LT ADC** -> **LT TCU**: "Heading 150 Visual, ABC"  
+    **LT ADC** -> **ABC**: "ABC, Assigned heading right 150 Visual, Runway 14L, Cleared for Takeoff"  
+    **ABC** -> **LT ADC**: "Right heading 150 Visual, Runway 12, Cleared for Takeoff, ABC"  
+    `AIP GEN 3.4`
 
-The Standard Assignable level from HB ADC to HBA is:  
-For Jets: `A080`  
-For Non-Jets: The lower of `A045` or the `RFL`.
-### Arrivals
-LTA will coordinate all arrivals into Launceston Prior to 5 mins from the boundary.
+The LTA controller can suspend/resume Auto Release at any time, with the concurrence of **LT ADC**.
```

Comment:
No auto release out of LT?

Reply:
see no reason why we cant have it on for assigned standard assignable and SID